src/cli/commands/init.ts

```diff
@@ -226,16 +230,93 @@ async function promptConfigSettings(disablePrompts: boolean, detectedConfig: Fra
       initial: detectedConfig.appDevserverUrl,
       format: trimValue,
     },
+    {
+      type: null,
+      name: "apiLanguage",
+    },
+    {
+      type: null,
+      name: "apiVersion",
+    },
   ]);
+
+  if (response.apiLocation) {
```

Comment:
This is not how it should be done using prompts lib: the prompts should be set dynamically using using:

`type: (previous, response) => response.apiLocation ? 'select' : null`

Same for apiVersion.

Any prompt property can be dynamic, see: https://github.com/terkelg/prompts#dynamic-prompts